src/Evolution/DiscontinuousGalerkin/VariableFixing/RadiallyFallingFloor.hpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/Tags.hpp"
+#include "Utilities/EqualWithinRoundoff.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// Contains all variable fixers.
+namespace VariableFixing {
+namespace CacheTags {
+struct MaskRadius {
```

Comment:
You should someone link this to RadiallyFallingFloor, maybe a `\see` comment there? (Please verify that it renders correctly ;) )